src/qibolab/qubits.py

```diff
@@ -38,14 +38,25 @@ class Qubit:
     name: QubitId
 
     bare_resonator_frequency: int = 0
-    readout_frequency: int = 0  # this is the dressed frequency
+    readout_frequency: int = 0
+    """ Readout dressed frequency"""
     drive_frequency: int = 0
     anharmonicity: int = 0
-    Ec: int = 0
-    Ej: int = 0
-    g: int = 0
+    sweetspot: float = 0.0
+    flux_to_bias: float = 0.0
+    asymmetry: float = 0.0
+    brf: float = 0.0
```

Comment:
@DavidSarlle here can't you use `bare_resonator_frequency` at line 40? Or is it a different quantity?

Reply:
Nevermind I just read the update docstring

Reply:
https://github.com/qiboteam/qibolab/pull/581#discussion_r1332795116